src/pages/docs/platform/auth/capabilities.mdx

```diff
@@ -0,0 +1,897 @@
+  /* ... issue the TokenRequest to a client ... */
+```
+
+```rest_csharp
+  AblyRest rest = new AblyRest(new ClientOptions {Key = "{{API_KEY}}"});
+  TokenParams tokenParams = new TokenParams {ClientId = "Bob"};
+  string tokenRequest = await rest.Auth.CreateTokenRequestAsync(tokenParams);
+  // ... issue the TokenRequest to a client ...
+```
+
+```rest_objc
+  ARTRest *rest = [[ARTRest alloc] initWithKey:@"{{API_KEY}}"];
+  ARTTokenParams *tokenParams = [[ARTTokenParams alloc] initWithClientId:@"Bob"];
+  [rest.auth createTokenRequest:tokenParams options:nil
+    callback:^(ARTTokenRequest *tokenRequest, NSError *error) {
+      // ... issue the TokenRequest to a client ...
+  }];
+```
+
+```rest_swift
+  let rest = ARTRest(key: "{{API_KEY}}")
+  let tokenParams = ARTTokenParams(clientId: "Bob")
+  rest.auth.createTokenRequest(tokenParams, options: nil) { tokenRequest, error in
+    // ... issue the TokenRequest to a client ...
+  }
+```
+
+```rest_go
+rest, _ := ably.NewREST(
+  ably.WithKey("{{API_KEY}}"))
+params := &ably.TokenParams{
+  ClientID: "Bob",
+}
+tokenRequest, _ := rest.Auth.CreateTokenRequest(params)
+```
+
+```rest_flutter
+final rest = ably.Rest(options: ably.ClientOptions(key: '{{API_KEY}}'));
+final tokenRequest = await rest.auth.createTokenRequest(
+  tokenParams: ably.TokenParams(clientId: 'Bob'),
+);
+```
+</Code>
+
+#### Wildcard token auth <a id="wildcard"/>
+
+You can use [token authentication](/docs/platform/auth/authentication#token-based-authentication) to set a wildcard `clientId` using a value of `*` when creating a token. Clients are then able to assume any identity in their operations, such as when publishing a message or entering presence.
+
+### Unidentified clients <a id="unidentified"/>
+
+If no `clientId` is provided when using [token authentication](/docs/platform/auth/authentication#token-based-authentication) then clients are not permitted to assume an identity and will be considered an unidentified client in all operations. Messages published will contain no `clientId` and those clients will not be permitted to enter the [presence](/docs/presence-occupancy/presence) set.
+
+## Capabilities
+
+API keys and Ably-compatible tokens, have a set of capabilities assigned to them that specify which operations (such as subscribe or publish) can be performed on which channels.
+
+API keys are long-lived, secret and typically not shared with clients. API key capabilities are configured using the [dashboard](https://ably.com/dashboard), or using the [Control API](/docs/platform/account/control-api).
+
+Ably-compatible tokens are designed to be shared with untrusted clients, are short-lived, and can be configured and issued programmatically. See [selecting an authentication mechanism](/docs/platform/auth/authentication#how-to-choose-authentication-method) to understand why token authentication is the preferred option in most scenarios.
+
+### Resource names and wildcards <a id="wildcards"/>
```

Comment:
These bullets would greatly benefit from some extra grouping, be it just a sentnece ot a heading. For example, "Wildcard rules:".